homeassistant/components/nextdns/button.py

```diff
@@ -53,4 +57,22 @@ def __init__(
 
     async def async_press(self) -> None:
         """Trigger cleaning logs."""
-        await self.coordinator.nextdns.clear_logs(self.coordinator.profile_id)
+        try:
+            await self.coordinator.nextdns.clear_logs(self.coordinator.profile_id)
+        except (
+            ApiError,
+            ClientConnectorError,
+            TimeoutError,
+            ClientError,
+        ) as err:
+            raise HomeAssistantError(
+                translation_domain=DOMAIN,
+                translation_key="method_error",
+                translation_placeholders={
+                    "method": "clear_logs",
+                    "entity": self.entity_id,
+                    "error": repr(err),
+                },
```

Comment:
Hmm, there are only 2 methods used right? Would it make sense to make them specific?

Reply:
Removed

Reply:
Or maybe you meant creating separate types of translations? My thinking is slowly today 😄

Reply:
I don't mind either way, I think the error will explain it well enough what service it is, but we shouldn't hardcode the english string for non-exceptions